### DISCUSSION TITLE
autostart rustdesk on linux error "Failed to open URI"

### DISCUSSION THREAD

**Discussion by Mikesco3** [rustdesk/rustdesk#5739] [General]
I had successfully deployed a selfhosted rustdesk server, and I'm currently able to run the client on my linux (**Manjaro XFCE 4**). <br>
Now wanted to have it auto-start when I login to my linux machine.  <br>
I to added this to my session login: 
 `/usr/bin/rustdesk --tray`  <br>
However it doesn't seem to be able to listen for connections and when I right-click to open up the app, I get the following error:
>Failed to Open URI.
>The specified location is not supported.
> rustdesk://

I did the usual, google, read pending issues and haven't come across anything that might lead to a solution yet.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/releases/download/1.2.2/rustdesk-1.2.2-0-x86_64.pkg.tar.zst?

**Comment by Mikesco3** [comment]
I'm running version 1.2.2 
Fingerprint: 3673 c523 3d2a 8e2d 870a 69a2 0988 a063 a172 1533 56b3 5f42 a798 1937 dba7 8201
